Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -24,4 +27,38 @@ public async Task<string> GetWordOfDay(double offsetInHours = -7.0)
         DateOnly today = DateOnly.FromDateTime(DateTime.UtcNow.AddHours(offsetInHours));
         return await wordOfTheDayService.GetWordOfTheDay(today);
     }
+
+    [HttpGet("WordOfTheDay/{date}")]
+    public async Task<string> GetWordOfDay(DateTime date)
+    {
+        DateOnly dateOnly = DateOnly.FromDateTime(date);
+        return await wordOfTheDayService.GetWordOfTheDay(dateOnly);
+    }
+
+    [HttpGet("WordsList/")]
+    public async Task<WordResultDto> GetWordsList(string query = "", int page = 1, int pageSize = 10)
+    {
+        return await wordOfTheDayService.GetWordsList(query, page, pageSize);
+    }
+
+    [HttpPost("AddWord")]
+    [Authorize(Policy = Policies.EditWord)]
+    public async Task AddWord(WordDto word)
+    {
+        await wordEditorService.AddWord(word);
+    }
+
+    [HttpDelete("DeleteWord")]
+    [Authorize(Policy = Policies.EditWord)]
+    public async Task DeleteWord(string word)
+    {
+        await wordEditorService.DeleteWord(word);
+    }
+
+    [HttpPost("EditWord")]
+    [Authorize]
+    public async Task EditWord(WordDto word)
+    {
```

Comment:
Controllers all look great for the word Editor!

Reply:
Thank you 😄